### PR TITLE
general improvements/cleanup

### PR DESCRIPTION
**Pull request by DioEgizio**

### CODE REVIEW

File: src/wiki/getting-started/install-of-alternatives.md [hunk 1]
```diff
@@ -33,12 +33,11 @@ If you would like to go a bit further, LamdAurora maintains a very detailed list
 
 # Fabulously Optimized
 
-If you don't want to search, and install these mods manually, then the [Fabulously Optimized](https://github.com/Fabulously-Optimized/fabulously-optimized) modpack is for you!
+If you don't want to search, and install these mods manually, then give a try to the [Fabulously Optimized](https://github.com/Fabulously-Optimized/fabulously-optimized) modpack.
```
Comment (txtsd) [inline src/wiki/getting-started/install-of-alternatives.md:36]: `give a try to` doesn't sound right :eyes:

Comment (DioEgizio) [reply src/wiki/getting-started/install-of-alternatives.md:36]: how should it be then lol

Comment (txtsd) [reply src/wiki/getting-started/install-of-alternatives.md:36]: Just `try` :sweat_smile: 
`...then try the...`

Comment (DioEgizio) [reply src/wiki/getting-started/install-of-alternatives.md]: done

**Review by txtsd — APPROVED** [review]

**Pull request merged**